python-packages/sra_client/sra_client/__init__.py

```diff
@@ -2,6 +2,143 @@
 
 # flake8: noqa
 
+"""Demonstration of using the python sra_client.
+
+**Post Order**
+
+Post an order to a SRA compliant 0x-relayer.
+
+
+>>> import sra_client
+>>> example_signed_order = {
+...     "makerAddress": "0x5409ed021d9299bf6814279a6a1411a7e866a631",
+...     "takerAddress": "0x0000000000000000000000000000000000000000",
+...     "senderAddress": "0x0000000000000000000000000000000000000000",
+...     "exchangeAddress": "0x35dd2932454449b14cee11a94d3674a936d5d7b2",
+...     "feeRecipientAddress":
+...         "0x0000000000000000000000000000000000000000",
+...     "makerAssetData": "0xf47261b0000000000000000000000000"
+...         "d0a1e359811322d97991e03f863a0c30c2cf029c",
+...     "takerAssetData": "0xf47261b0000000000000000000000000"
+...         "2002d3812f58e35f0ea1ffbf80a75a38c32175fa",
+...     "salt": "2362734632784682376287462",
+...     "makerFee": "0",
+...     "takerFee": "0",
+...     "makerAssetAmount": "1000000000000000000",
+...     "takerAssetAmount": "500000000000000000000",
+...     "expirationTimeSeconds": "999999999999999999999",
+...     "signature": (
+...         "0x1cb085506ccef3d15061766808a6d5b5369a6dacc323101f704ab1b38d0166725"
+...         "002379d576b1ddffee6adcfc080ff7118d20beae723d3708ce4e04e49dd92694003")
+... }
+>>> relayer_api = sra_client.api.default_api.DefaultApi()
```

Comment:
We talked about changing the client construction so that a URL is a required parameter.  More specifically, I think these demos should show how a URL is passed in, so I'd like to see the `http://localhost:3000` explicitly passed in in all of the demos.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1734/commits/3bbc3411f03eb9a921e93779e7b00e9a5b2c4b4d